flink-staging/flink-ml/src/test/scala/org/apache/flink/ml/preprocess/StandardizerSuite.scala

```diff
@@ -0,0 +1,189 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.flink.ml.preprocess
+
+import org.apache.flink.api.scala._
+import org.apache.flink.ml.math.{Vector, DenseVector}
+import org.scalatest._
+
+
+class StandardizerSuite extends FlatSpec with Matchers {
+  behavior of "Flink's Standard Scaler"
+  import StandardizerData._
+  it should "first center and then properly scale the given vectors" in {
+
+    val env = ExecutionEnvironment.getExecutionEnvironment
+    env.setParallelism(2)
```

Comment:
This will be done by the ```FlinkTestBase``` for you. Only if you have to set it to a specific value, then override the inherited ```parallelism``` value from ```FlinkTestBase```.